### PR TITLE
Setp site max-width to 1200px

### PR DESCRIPTION
**Pull request by ftynse**
This significantly improves readability on wide and large screens by removing the need for the reader to scan the entire screen and concentrating the whitespace on the edges. This is consistent with the best practices adopted by Google search results, Github, and multiple press websites that are text-heavy, similarly to documentation websites.

### CODE REVIEW

**Comment by ftynse** [comment]
Before and after from a 4k screen.

Screenshot from 2022-12-05 16-10-16
Screenshot from 2022-12-05 16-10-07

File: website/themes/mlir/src/scss/_structure.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@ body {
 
 .container {
 	width: 100%;
+        max-width: 1200px;
```
Comment (joker-eph) [inline website/themes/mlir/src/scss/_structure.scss:11]: Indentation is funky here.

**Comment by ftynse** [comment]
Ping

**Review by jpienaar — APPROVED** [review]

**Comment by joker-eph** [comment]
I find this new layout not totally well set up: that is I understand we don't want the content to spread on the full width of the screen, but if you look at layouts like GitHub for example, you still have the top level menu bar spreading and some graphical elements taking this into account. 
Right now there is also a slight color different between the various backgrounds: 0xFA for the page background but 0xFF for the header background and  0XF9 for the menu column background. These were intended while there wasn't a border to the left between these elements (and to the right for the header) but they don't fit now.

**Comment by ftynse** [comment]
Background colors can be trivially changed. (I don't perceive the difference between 0xFA and 0xFF).

I don't think spreading works here. It doesn't really look very nice for Github either on 4k, but they can at least balance the weight of the page by having components in both top-left and top-right corners. We don't have that.

one
two
three

**Comment by ftynse** [comment]
The top menu may feel distracting, maybe we can change its background to use a less bold color. 

I also don't fully understand why do we need two partially overlapping menus. I doubt somebody needs to navigate to the youtube channel landing from dialect documentation. The "Bugs" link in the top menu still points to Bugzilla without anybody complaining :)

**Pull request merged**